src/Front/Identifiers.hs

```diff
@@ -0,0 +1,25 @@
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
This module is where we will put information on reserved/builtin identifiers. (oops I forgot documentation, will add)